### PR TITLE
feat: provide docker support for portal

### PR DESCRIPTION
**Pull request by xixas**
- Added support for docker for portal.
- Updated document for using portal with docker.

Reference for why dockerizing using Nginx
https://serengetitech.com/tech/how-to-dockerize-your-angular-14-application/

### CODE REVIEW

File: apps/portal/.env.example [hunk 1]
```diff
@@ -0,0 +1 @@
+SERVER_PORT=5000
```
Comment (osm-vishnukyatannawar) [inline apps/portal/.env.example:1]: when we dockerize this do we have control over API URL? Which API url should this portal use?

**Review by osm-vishnukyatannawar — CHANGES_REQUESTED** [review]

File: apps/portal/.env.example (hunk 1 above)
Comment (xixas) [reply apps/portal/.env.example:1]: No, API end point should already be in repository inside `environment.prod.ts` file
(which ever is configured as production in angular.json)

Comment (osm-vishnukyatannawar) [reply apps/portal/.env.example:1]: https://github.com/OsmosysSoftware/osmo-notify/blob/main/apps/portal/src/environments/environment.prod.ts#L5

image


I see this right now. So it won;t work right if we deploy as docker on docker tower.

Comment (xixas) [reply apps/portal/.env.example:1]: No, it should have been production API in environment.prod.ts

Comment (xixas) [reply apps/portal/.env.example:1]: image
image

Comment (osm-vishnukyatannawar) [reply apps/portal/.env.example:1]: My question is why not add a commit since we already know the production url like we did in JLR.

Comment (xixas) [reply apps/portal/.env.example:1]: Okay will add here itself.

Comment (xixas) [reply apps/portal/.env.example:1]: Updated production url.
To Check
1. Production Backend is updated with latest GraphQL API.
2. Check for CORS Error if present.

**Review by osm-vishnukyatannawar — APPROVED** [review]

**Pull request merged**